jetty-server/src/main/java/org/eclipse/jetty/server/Connector.java

```diff
@@ -102,4 +104,48 @@
      * @return The connector name or null.
      */
     String getName();
+
+    /**
+     * Get the Local Address of the connection
+     *
+     * @return Returns the connection local address.
+     */
+    SocketAddress getLocalAddress();
+
+    /**
+     * <p>
+     * Specify the connection local address used within application API layer
+     * when identifying the local host name/port of a connected endpoint.
+     * </p>
+     * <p>
+     * This allows an override of higher level APIs, such as
+     * {@code ServletRequest.getLocalName()}, {@code ServletRequest.getLocalAddr()},
+     * and {@code ServletRequest.getLocalPort()}.
+     * </p>
+     *
+     * @param localAddress the address to use for host/addr/port, or null to reset to default behavior
+     */
+    void setLocalAddress(SocketAddress localAddress);
+
+    /**
+     * Get the optional Server URI authority default
+     *
+     * @return Returns the connection server authority (name/port).
```

Comment:
```suggestion
     * @return Returns the connection server authority (name/port) or null.
```

I really don't like this "or null".